.codegen/model.go.tmpl

```diff
@@ -79,7 +79,7 @@ func (f *{{.PascalName}}) Type() string {
 	{{- else if .ArrayValue }}[]{{template "type" .ArrayValue}}
 	{{- else if .MapValue }}map[string]{{template "type" .MapValue}}
 	{{- else if .IsObject }}{{.PascalName}}
-	{{- else if .Enum }}{{.PascalName}}
+	{{- else if .Enum }}types.String
```

Comment:
Just to confirm -- are all enums always strings? or can some be int as well? I took a look at few of them and those seem to be string but just want to confirm if we have a constraint somewhere that enums are strings only.

Reply:
from looking at the codegen template for the gosdk structs, yes, that seems to be the case